<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-graphics"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="insertNode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>515</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserir nó</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nodeName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>463</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>517</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Mudar rótulo</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="newName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showMatrix">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>538</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz adj.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="routes">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>517</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotas</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="src">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>473</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="dst">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>473</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>473</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>O:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>473</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>D:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sortCost">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Custo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sortLen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>530</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Distância</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Organizar:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="weightV">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>515</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Vis. peso</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="weightP">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pop-up peso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="complete">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Completar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="insertAmount">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>523</y>
      <width>38</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>505</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Qtd:</string>
    </property>
   </widget>
   <widget class="QLabel" name="insertionEr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string>*Rótulo já existe</string>
    </property>
   </widget>
   <widget class="QLabel" name="insertionEr1">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>560</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string>*Rótulo já existe</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enDirection">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Direção</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enWeight">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>485</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Peso</string>
    </property>
   </widget>
   <widget class="QLabel" name="routeEr">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>570</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matrixEr">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>570</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string>*Insira pelo menos 2 nós</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
